src/main/java/com/softdevsix/domain/files/enums/Severity.java

```diff
@@ -0,0 +1,4 @@
+package com.softdevsix.domain.files.enums;
+
+public enum Severity {
```

Comment:
This enum already exists in another directory, and this enum isn't complete, erase this class